src/anim/controller/anim-blend-tree.js

```diff
@@ -48,6 +50,10 @@ class AnimBlendTree extends AnimNode {
         return this._parent ? this._parent.weight * this._weight : this._weight;
     }
 
+    get syncAnimations() {
+        return this._syncAnimations;
+    }
+
     getChild(name) {
```

Comment:
it probably is not expected to happen .. but should there be `return null;` outside the loop?

Reply:
this comment is for ` getChild(name) {` function, not syncAnimations as displayed here .. in diff this is displayed more correctly.